src/wh_server_crypto.c

```diff
@@ -251,6 +251,9 @@ static int wh_Server_HandleRsaFunction(whServerContext* server, whPacket* packet
     RsaKey rsa[1] = {0};
 
     int op_type     = (int)(packet->pkRsaReq.opType);
+    uint32_t options = packet->pkRsaReq.options;
+    int evict       = options & WH_PACKET_PK_ECCSIGN_OPTIONS_EVICT;
```

Comment:
nano-nit: I like `!!` to be explicit that this is turning it into a boolean flag

Reply:
updated.